internal/cri/server/sandbox_stats_linux.go

```diff
@@ -76,14 +83,17 @@ func (c *criService) podSandboxStats(
 		if err != nil {
 			return nil, fmt.Errorf("failed to obtain network stats: %w", err)
 		}
+		defaultInterface := &runtime.NetworkInterfaceUsage{
+			Name:     defaultIfName,
+			RxBytes:  &runtime.UInt64Value{Value: linkStats.RxBytes},
+			RxErrors: &runtime.UInt64Value{Value: linkStats.RxErrors},
+			TxBytes:  &runtime.UInt64Value{Value: linkStats.TxBytes},
+			TxErrors: &runtime.UInt64Value{Value: linkStats.TxErrors},
+		}
 		podSandboxStats.Linux.Network = &runtime.NetworkUsage{
-			DefaultInterface: &runtime.NetworkInterfaceUsage{
-				Name:     defaultIfName,
-				RxBytes:  &runtime.UInt64Value{Value: linkStats.RxBytes},
-				RxErrors: &runtime.UInt64Value{Value: linkStats.RxErrors},
-				TxBytes:  &runtime.UInt64Value{Value: linkStats.TxBytes},
-				TxErrors: &runtime.UInt64Value{Value: linkStats.TxErrors},
-			},
+			Timestamp:        timestamp.UnixNano(),
+			DefaultInterface: defaultInterface,
+			Interfaces:       []*runtime.NetworkInterfaceUsage{defaultInterface},
```

Comment:
This should not contain the network usage from the default interface. Ref: https://github.com/kubernetes/cri-api/blob/master/pkg/apis/runtime/v1/api.proto#L765-L766